symExec.py

```diff
@@ -1299,13 +1307,15 @@ def sym_exec_ins(start, instr, stack, mem, global_state, path_conditions_and_var
     elif instr_parts[0] == "SHA3":
         if len(stack) > 1:
             global_state["pc"] = global_state["pc"] + 1
-            stack.pop(0)
-            stack.pop(0)
+            s0 = stack.pop(0)
+            s1 = stack.pop(0)
+            tmp = [str(x) for x in memory[s0: s0 + s1]]
```

Comment:
Yeah. This PR is not complete. This SHA3 works with my tests when it's used to find a key in the storage. Other SHA3 should be handled differently as s0 and s1 could be symbolic